sharktank/sharktank/layers/mixture_of_experts_block.py

```diff
@@ -36,11 +36,18 @@ def __init__(
         score_experts=softmax,
         normalize_experts=True,
         add_residual=True,
+        tensor_parallel_devices: Optional[tuple[int, ...]] = None,
```

Comment:
I am actually thinking to drop this init parameter and get this from forward's input.

Reply:
I think that's a good idea. The extra state isn't needed.

Reply:
I removed it.